backend/src/main/java/de/adrianaschepers/shishabuddies/controller/UserController.java

```diff
@@ -61,6 +66,72 @@ public UserController(UserService userService) {
         return ok(createdSettings);
     }
 
+    @PostMapping("create-setup")
+    public ResponseEntity<Setup> createSetup(@RequestBody Setup setup, @AuthenticationPrincipal UserEntity authUser){
+        SetupEntity setupEntity = map(setup);
+        SetupEntity createdSetupEntity = userService.saveSetup(setupEntity,authUser);
+        Setup createdSetup = map(createdSetupEntity);
+        return ok(createdSetup);
+    }
+
+    @GetMapping("all-setups")
+    public ResponseEntity<List<Setup>> getAllSetups(@AuthenticationPrincipal UserEntity authUser){
+        List<SetupEntity> setups = userService.getAllSetups(authUser);
+        if(setups==null){
+            return notFound().build();
+        }
+        return ok(mapSetup(setups));
+    }
+
+    @GetMapping("{title}/setup")
```

Comment:
Wahrscheinlich Geschmacks-Sache, aber ich würde immer zuerst die fixen werte und danach die variablen in die url packen, also "setup/{title}"